src/utils.ts

```diff
@@ -132,3 +132,16 @@ export const isDefined = (value: any): boolean => value !== void 0;
 
 export const isMatchingPath = (path: string, criteria: string | RegExp) =>
     isRegex(criteria) ? !!path.match(criteria as RegExp) : checkGlob(path, criteria as string);
+
+export const isExpanded = (path: string, state: JsonViewerState): boolean => state.expanded[path];
```

Comment:
I had that originally, but I was sort of shooting for not adding any functions exposed on the `JsonViewer` to try to reduce impact on the current library. I also assumed since the author was already importing from `./types` then they weren't concerned with bleeding app specific code in here. I'm open to moving them out, but does the main class make the most sense?

Reply:
to me, yeah, but we can leave it to the author to opine on it